ui/src/app/views/workspace-share/component.spec.ts

```diff
@@ -141,101 +149,75 @@ describe('WorkspaceShareComponent', () => {
         WorkspaceShareComponent
       ],
       providers: [
-        { provide: WorkspacesService, useValue: new WorkspacesServiceStub() },
-        { provide: ActivatedRoute, useValue: activatedRouteStub },
-        { provide: ProfileStorageService, useValue: new ProfileStorageServiceStub() },
+        { provide: UserService, useValue: new UserServiceStub()},
+        { provide: WorkspacesService, useValue: new WorkspacesServiceStub()},
+        { provide: ActivatedRoute, useValue: activatedRouteStub},
+        { provide: ProfileStorageService, useValue: new ProfileStorageServiceStub()},
         {
           provide: ServerConfigService,
           useValue: new ServerConfigServiceStub({
             gsuiteDomain: 'fake-research-aou.org'
           })
         }
-      ]}).compileComponents().then(() => {
-        workspaceSharePage = new WorkspaceSharePage(TestBed);
-        workspaceSharePage.fixture.componentRef.instance.profileStorageService.reload();
+      ]
+    }).compileComponents().then(() => {
+      workspaceSharePage = new WorkspaceSharePage(TestBed);
+      workspaceSharePage.fixture.componentRef.instance.accessLevel = WorkspaceAccessLevel.OWNER;
```

Comment:
Nit: You should just be able to use `workspaceSharePage.fixture.componentInstance`